<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaselineSearchReplaceDialog</class>
 <widget class="QDialog" name="BaselineSearchReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Baseline Search-and-Replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="WritingSystemCombo" name="writingSystemCombo"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Replace:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LingEdit" name="searchEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="searchLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>With:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LingEdit" name="replaceEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="replaceLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WritingSystemCombo</class>
   <extends>QComboBox</extends>
   <header location="global">writingsystemcombo.h</header>
  </customwidget>
  <customwidget>
   <class>LingEdit</class>
   <extends>QLineEdit</extends>
   <header>lingedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BaselineSearchReplaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BaselineSearchReplaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
